lib/managers/files.js

```diff
@@ -1383,6 +1393,21 @@ Files.prototype.getRepresentationInfo = function(fileID, representationType, cal
 			// 200 - A Boxfile representation generated successfully
 			// return the representation object
 			case httpStatusCodes.OK:
+
+				if (options && options.fullInfo) {
+
+					var data = response.body.representations.entries;
+					var promiseArray = data.map(entry => {
+						if (entry.status.state === 'success' || entry.status.state === 'viewable') {
```

Comment:
This should probably also include the `"error"` state, which isn't something we should poll on.  Might be worth adding a small test where one of the reps is errored and define the behavior for that case.